src/Carousel.vue

```diff
@@ -530,6 +565,27 @@ export default {
       }
       return this.carouselWidth;
     },
+    /**
+     * Get the maximum height of the carousel active slides
```

Comment:
This is semi-misleading, I think we should say `Sets the maximum height...` rather than `Gets`. We don't actually return a value here either, so we can remove the return type